src/main/java/com/hotelservice/hotelApi/constant/CommonExceptionStatus.java

```diff
@@ -0,0 +1,15 @@
+package com.hotelservice.hotelApi.constant;
+
+public enum CommonExceptionStatus {
+    DEFAULT,
+    INTERNAL_SERVER_ERROR,
+    HOTEL_NOT_FOUND,
```

Comment:
NO need exception for each case, just  NOT_FOUND, NOT_VALID - it`s enough, add a description